src/convertToObject.js

```diff
@@ -12,7 +12,13 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return Object.fromEntries(
```

Comment:
Wow! Very nice solution. Thanks for idea)

Reply:
Thank U for the review)